src/AssetManager.cpp

```diff
@@ -0,0 +1,95 @@
+#include "AssetManager.h"
+
+#include "assimp/Importer.hpp"
+#include "assimp/scene.h"
+#include "assimp/postprocess.h"
+
+#include "glm/gtc/type_ptr.hpp"
+
+#include <iostream>
+
+AssetManager::AssetManager() {
```

Comment:
Is there any benefit to the empty constructor over the implicit?

Reply:
I'd agree with having an explicit one, i think it's easier to read even if it's just a little bit.